### ISSUE TITLE
scrape data from other sources

### ISSUE THREAD

**Issue by berteh** [jwnigel/permaculture#5]
Would love to be able to collect+merge existing sources and provide a means for a community of users to curate/improve it.

That collaborative evolution process does not need to be in the app: this 'reference' data could live somewhere online (along with translations, tips, anecdotes, pictures, seeds/plants local providers coordinates, or feedback) and be exported to the app on new release or manual update.

Sources already considered for import/inspiration:

- [x] https://pfaf.org
- [x] https://permapeople.org plants database (and its many filters) **This one I still need to scrape

Sources for specific fruit cultivars:
- [x] https://orangepippin.com
- [ ] https://pomiferous.com (Even more thorough than orangepippin
Others to include pears, stones fruits, etc...

Sources maybe interesting but not yet scrapped:
- [ ] Edible Forest plants set, hundreds of edible or useful plants, readily re-encoded at https://docs.google.com/spreadsheets/d/1_PgxV4pxlNTa0Ep9TDY_57oJfVtrofmu/edit#gid=1609043836$-
- [ ] roots systems : https://images.wur.nl/digital/collection/coll13/search/searchterm/wurzelatlas%20:%20mitteleuropaeischer%20ackerunkraeuter%20und%20kulturpflanzen/field/subcol/mode/all/conn/and/

Sources unlikely to be integrated in anyway, but that may be interesting for permadesigners, for info:
- http://www.gartendatenbank.de/wiki/ along with good gardening tips
- https://www.plantopedia.de/pflanzenbestimmung/ and their plant recognition indexes based on observable characteristics
- https://alveoles.fr/fiches-plantes for their beautiful paper cards. Their online database provides good info, though no sexy looks: https://plantes.universite-alveoles.fr and lacking some filters to find good companion plants.

**Comment by jwnigel** [comment]
Thanks for these. I am not familiar with any of them! 

The permapeople db looks very good--I don't have any immediate ideas on how to combine different databases, like which how to merge two different entries, but it would be good to cast a wide net for sources. 

I don't know if you're familiar with the books Edible Forest Gardens? There is a matrix--basically a printed database--at the end that includes many hundreds of edible or useful plants. Someone replicated the entire thing and [shared it on reddit](https://docs.google.com/spreadsheets/d/1_PgxV4pxlNTa0Ep9TDY_57oJfVtrofmu/edit#gid=1609043836).

**Comment by berteh** [comment]
I am familiar with the book, but didn't know about the re-encoded table, thanks!

About merging data... maybe simply check matching latin name? If none it's new, if there is purpose a manual 'diff-like' merge dialog, where the user would be supposed to select between redundant fields?

That work of date import/refine could also happen outside of the tool in a community-driven collaborative date space (online spreadsheet or database or other)... and the tool would only query that data source for plants and maybe allow to refine some (eg as a 'local' sub/variety) or suggest a new one to be curated for the central repository.

I quite like the idea of being able to quickly work with 'my' varieties, but still being able to contribute to a better global dataset... that would still allow to be particular to many local climates/altitude/conditions...

**Comment by jwnigel** [comment]
Yes, these are great ideas (Sorry I just saw this). I am going to first work on scraping permapeople.org and then we can try merging it with the pfaf data.

**Comment by berteh** [comment]
no rush on this point. we will only need it much later.

**Comment by berteh** [comment]
adding nice potential source for roots systems:
https://images.wur.nl/digital/collection/coll13/search/searchterm/wurzelatlas%20:%20mitteleuropaeischer%20ackerunkraeuter%20und%20kulturpflanzen/field/subcol/mode/all/conn/and/

if you have good contacts to researchers maybe you could ask for a download of data at https://research.wur.nl/en/datasets/coded-figure-captions-of-the-wurzelatlas-mitteleuropaeischer-acke
